### ISSUE TITLE
Window jumping to original position when dragging

### ISSUE THREAD

**Issue by yusi1** [xmonad/xmonad#290]
### Problem Description

While moving a window by dragging the mouse and holding alt, the window jumps back to the original position.
(resizing the window / floating window works)
Related:
https://github.com/xmonad/xmonad/pull/64

### Configuration File
Below

### Checklist

  - [Y] I've read [CONTRIBUTING.md](https://github.com/xmonad/xmonad/blob/master/CONTRIBUTING.md)

  - [Y] I tested my configuration with [xmonad-testing](https://github.com/xmonad/xmonad-testing)

**Comment by liskin** [comment]
Now why would you think providing your config file is "not applicable" here? :-/

**Comment by yusi1** [comment]
https://github.com/yusi1/my_dotfiles/blob/main/.xmonad/xmonad.hs

**Comment by yusi1** [comment]
This issue occurred when I commented updatePointer out aswell

**Comment by yusi1** [comment]
Same issue occurred also when I changed my mouse to another one I had lying around, same occurred with a different keyboard

**Comment by yusi1** [comment]
Although I did update the kernel of my system, that could of caused this possibly

**Comment by slotThe** [comment]
Is this on all windows, or only some?  I know telegram _really_ wants its windows to be fullscreen and at _exactly_ `RationalRect 0 0 1 1`, otherwise you're going to have a bad time (I haven't had the energy to track this down and decide who to shout at yet)

**Comment by yusi1** [comment]
> Is this on all windows, or only some? I know telegram _really_ wants its windows to be fullscreen and at _exactly_ `RationalRect 0 0 1 1`, otherwise you're going to have a bad time (I haven't had the energy to track this down and decide who to shout at yet)

Unfortunately, this seems to be happening to all windows for me. The reason I thought that I would post the issue in the XMonad GitHub is that I saw a pull request from a few years ago, that was "related" I guess? I didn't know where to post this either to be honest.

**Comment by yusi1** [comment]
Hmm, it seems when trying another window manager (QTile is what I tried this time) the same issue occurs when trying to drag floating windows, so this rules out that XMonad is the culprit, could it be an issue at the hardware level?, this is very odd.

**Comment by geekosaur** [comment]
Hardware level would be extremely odd. It sounds like an application "fighting back" against a window move, but that shouldn't affect all windows.

**Comment by yusi1** [comment]
Here's a video of what happens for more context:
https://imgur.com/a/4nLSGqD

**Comment by yusi1** [comment]
Fixed! it was the compositor  (picom-jonaburg-git). I was using an unofficial fork of picom that was buggy, I had animations turned on in the configuration file and they were causing this specifically. I switched back to the actual picom and this issue was gone.

Since I was using XMonad and QTile on the same user, the same .xprofile would be used (since it's in the home directory) so they were both using the same compositor launched through .xprofile (command in .xprofile was: picom -b & disown), this is why the issue occurred and would of occurred on any window manager I tried on the user.
